docs/cheatsheet.rst

```diff
@@ -4,68 +4,10 @@ Cheatsheet
 
 .. index:: precedence
 
```

Comment:
I'd either remove this or at least use the same index entry as in the other place (without `!` though since it's not the main entry on the topic).
```suggestion
.. index:: operators; precedence
```